proposal.rst

```diff
@@ -1,25 +1,26 @@
 .. This document is written in reStructuredText, a simple and unobstrusive
 .. markup language.  For an introduction to reStructuredText see:
-.. 
+..
 .. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
-.. 
+..
 .. Lines like this which start with `.. ` are comments which won't appear
 .. in the generated output.
-.. 
+..
 .. To apply for a GSoC project with Xapian, please fill in the template below.
 .. Placeholder text for where you're expected to write something says "FILLME"
 .. - search for this in the generated PDF to check you haven't missed anything.
-.. 
+..
 .. See our GSoC Project Ideas List for some suggested project ideas:
 .. https://trac.xapian.org/wiki/GSoCProjectIdeas
 ..
 .. You are also most welcome to propose a project based on your own ideas.
-.. 
+..
 .. From experience the best proposals are ones that are discussed with us and
 .. improved in response to feedback.  You can share draft applications with
 .. us by forking the git repository containing this file, filling in where
 .. it says "FILLME", committing your changes and pushing them to your fork,
 .. then opening a pull request to request us to review your draft proposal.
+
 .. Please pick a useful title for the pull request - "My application" may
 .. be meaningful to you, but isn't helpful to those reviewing.  Using the
 .. title of the project is a good.  You can open a pull request even before
```

Comment:
"Draft Proposal" isn't a very useful title - it tells the reader nothing about the proposal, except that it's a draft (and at this stage all proposals are really still drafts).

Reply:
Hi @ojwb , apologies for the uninformative title. I’ve edited it now with the name of the project.